changelog.txt

```diff
@@ -6,6 +6,7 @@
 * Fix - Fixed unhandled php error when invalid card is used on Pay Order page.
 * Fix - Fixed bug that would send outdated data when using Blocks Checkout.
 * Add - List of payment methods that are not included for the merchant.
+* Tweak - Remove html from translatable strings
```

Comment:
Because WCPay 3.4.0 was released two days ago, we'd need to update this entry in the changelog.

Reply:
Fixed by 3d8acffcb01243cad22d9eb7b73a5dbadc18e1a0